src/Jackett.Common/Indexers/Wolfmax4K.cs

```diff
@@ -0,0 +1,498 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Html.Parser;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json.Linq;
+using NLog;
+using WebClient = Jackett.Common.Utils.Clients.WebClient;
+
+namespace Jackett.Common.Indexers
+{
+    [ExcludeFromCodeCoverage]
+    public class Wolfmax4K : IndexerBase
+    {
+        public override string Id => "wolfmax4k";
+        public override string Name => "Wolfmax 4k";
+        public override string Description => "Wolfmax 4k is a SPANISH public tracker for MOVIES / TV";
+        public override string SiteLink { get; protected set; } = "https://wolfmax4k.com/";
+        public override string[] AlternativeSiteLinks => new[]
```

Comment:
if there is only one domain then you don not need to include an alternative site links block ;-)